scripts/install_deps.sh

```diff
@@ -12,7 +12,8 @@ conda install -y regex pillow tqdm boto3 requests numpy h5py scipy matplotlib un
 conda install -y -c conda-forge librosa inflect
 
 pip install -q fastBPE sacremoses sentencepiece subword_nmt editdistance
-pip install -q visdom mistune filelock tokenizers packaging pandas
+pip install -q visdom filelock tokenizers packaging pandas
+pip install -q mistune==2.0.2
```

Comment:
why is this needed?

Reply:
I don't know if it's a short thought
Checked the following logs and ran a downgrade.
https://circleci.com/gh/pytorch/hub/2694

Reply:
There was no problem with the history of merged without errors until 2022, and the mistune version at that time was 2.0.4.

Reply:
A build error was displayed.
What do you think about modifying all libraries installed before a specific date to a version

+ If the torch version goes down to the 1.x version, does it cause problems with interaction with other org pages?

Reply:
Thanks for the typo fixes @cpprhtn. Do you mind reverting those install changes in this PR so that we can merge it? We can follow-up on the install issue on another PR. I see you already opened https://github.com/pytorch/hub/issues/333, thanks!

Reply:
Thank you for your review!